Block/Menu.php

```diff
@@ -374,4 +384,13 @@ private function renderNode($node, $level)
         return $this->nodeTypeProvider->render($type, $node->getId(), $level);
     }
 
+    public function _prepareLayout()
+    {
+        $this->setTemplate(
```

Comment:
can we set the template in the class constructor?

Reply:
No, in constructor we have empty $_template variable

Reply:
Why can't you set the field value to `Snowdog_Menu::menu.phtml`?

Reply:
I see no changes related to previous comments

Reply:
You're right sorry I assign 'Snowdog_Menu::menu.phtml' to $_template and template is set in construct